Sources/PostgresNIO/New/PSQLRow.swift

```diff
@@ -63,3 +62,13 @@ extension PSQLRow {
         return try T.decode(from: &cellSlice, type: column.dataType, format: column.format, context: context)
     }
 }
+
+extension PSQLRow {
+    func decode<T: PSQLDecodable>(column: String, as type: T.Type, file: String = #file, line: Int = #line) throws -> T {
+        try self.decode(column: column, as: type, jsonDecoder: JSONDecoder(), file: file, line: line)
+    }
+
+    func decode<T: PSQLDecodable>(column index: Int, as type: T.Type, file: String = #file, line: Int = #line) throws -> T {
```

Comment:
```suggestion
    func decodeColumn<T: PSQLDecodable>(at columnIndex: Int, as type: T.Type = T.self, file: String = #file, line: Int = #line) throws -> T {
```
Nit: makes the naming a bit more swiftier (is this even a word? :D)
Similar, `decode` where `column` is a `String` could be `decodeColumn(named:as:)` or just `decodeColumn(_:as:)` 

Will this be public API?